appengine-vmruntime/vmruntime/tests/wsgi_test.py

```diff
@@ -372,8 +392,13 @@ def test_concurrent_requests_have_independent_env(self):
         concurrent_request_should_proceed.set()
 
         # Wait for the thread to finish and examine the results.
-        response = future.get(5)  # This will raise an exception on timeout.
-        self.assertNotIn('ENVIRONMENT_MODIFIED', json.loads(response.data))
+        t.join(5) # This will raise an exception on timeout.
+        self.assertNotIn('ENVIRONMENT_MODIFIED', unmodified_os_env)
```

Comment:
All of the sets of unmodified_os_env appear to be of the form:

```
unmodified_os_env = dict(os.environ)
```

dict() will make copies, so I'm not sure this is asserting the state of os.environ.

Don't you want a reference to os.environ, instead of a copy, for this assertion?